Idea.js

```diff
@@ -6,13 +6,19 @@ class Idea {
     this.star = false;
   }
   saveToStorage(){
-
+    var stringifiedIdea = JSON.stringify(this);
+    window.localStorage.setItem(this.id, stringifiedIdea)
   }
 
   deleteFromStorage(){
-
+    window.localStorage.removeItem(this.id)
   }
-  updateIdea(){
 
+  updateIdea(){
+    if (!this.star) {
+      this.star = true;
+    } else {
+      this.star = false;
+    }
```

Comment:
I'm glad we got these all working in idea.js; it helps the code be much cleaner.